app/javascript/components/confirm-modal.js

```diff
@@ -0,0 +1,30 @@
+import { Modal } from "bootstrap";
+
+class ConfirmModal {
+  constructor() {
+    window.Turbo.setConfirmMethod(this.confirm.bind(this));
+    this.element = document.querySelector("#confirm-modal");
```

Comment:
Je comprends l'idée d'avoir fait ça, pour coller au reste du code js qu'on injecte dans l'appli (avant d'utiliser stimulus) mais je trouve que le fait d'utiliser une classe ici perturbe peut-être la compréhension de ce que fait cette classe et que peut-être qu'une simple fonction aurait été plus claire.

En tous cas je pense qu'à minima il faudrait renommer les variables `element`  qui est un argument de  `confirm` et `this.element` qui ne renvoient pas la même chose

Reply:
Oui effectivement t'as raison pour le element vs this.element je corrige 👍 , pour ce qui est d'utiliser une classe ça me semble relativement clair 🤷‍♂️